File: Java/Septiembre/Semana-18-al-22.java
// Ejercicio Jesús Mercado
// application.properties
# Conexión a MySQL
spring.datasource.url=jdbc:mysql://localhost:3306/tienda_libros_db?createDatabaseIfNotExist=true
spring.datasource.username=root
spring.datasource.password=admin
spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver

# Crear la base de datos en caso de ser necesario
spring.jpa.hibernate.ddl-auto=update
spring.jpa.show-sql=true

// TiendaLibrosApplication.java
package utn.tienda_libros;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class TiendaLibrosApplication {

	public static void main(String[] args) {
		SpringApplication.run(TiendaLibrosApplication.class, args);
	}

}

// Fin Ejercicio Jesús Mercado

//Ejercicio victoria Zaccaro

#Conexión a MySQL
spring.datasource.url=jdbc:mysql://localhost:3306/tienda_libros_db?createDatabaseIfNotExist=true
spring.datasource.username=root
spring.datasource.password=admin
spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver

#Crear la base de datos en caso de ser necesario
spring.jpa.hibernate.ddl-auto=update
spring.jpa.show-sql=true

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class TiendaLibrosApplication {
	public static void main(String[] args) {
		SpringApplication.run(TiendaLibrosApplication.class, args);
	}
}
//Fin ejercicip victoria zaccaro

//Ejercicio Gerardo Duckwitz
# Conexion a MySql
spring.datasource.url=jdbc:mysql://localhost:3306/tienda_libros_db?createDatabaseIfNotExist=true
spring.datasource.username=root
#spring.datasource.password=admin
spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver

#Crear la base de datos en caso de ser necesario
spring.jpa.hibernate.ddl-auto=update
spring.jpa.show-sql=true


package utn.tienda_libros;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class TiendaLibrosApplication {

	public static void main(String[] args) {
		SpringApplication.run(TiendaLibrosApplication.class, args);
	}

}

//Fin ejercicio Gerardo Duckwitz

//Ejercicio de Yesica López

# Conexión a MySQL
spring.datasource.url=jdbc:mysql://localhost:3306/tienda_libros_db?createDatabaseIfNotExist=true
spring.datasource.username=root
#spring.datasource.password=admin
spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver

#Crear la base de datos en caso de ser necesario
spring.jpa.hibernate.ddl-auto=update
spring.jpa.show-sql=true

package utn.tienda_libros;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class TiendaLibrosApplication{
	public static void main(String[] args){ SpringApplication.run(TiendaLibrosApplication.class, args);}
}

//Fin Ejercicio Yesica López

// Ejercicio Jose Remaggi
// application.properties
# Conexión a MySQL
spring.datasource.url=jdbc:mysql://localhost:3306/tienda_libros_db?createDatabaseIfNotExist=true
spring.datasource.username=root
spring.datasource.password=admin
spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver

# Crear la base de datos en caso de ser necesario
spring.jpa.hibernate.ddl-auto=update
spring.jpa.show-sql=true

// TiendaLibrosApplication.java
package utn.tienda_libros;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class TiendaLibrosApplication {

	public static void main(String[] args) {
		SpringApplication.run(TiendaLibrosApplication.class, args);
	}

}

// Fin Ejercicio Jose Remaggi
// Ejercicio Anabel Alesci
//Conexión a MySQL
spring.datasource.url=jdbc:mysql:localhost:3306/tienda_libros_db?createDatabaseIfNotExist=true
spring.datasource.username=root
spring.datasource.password=admin
spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver

//Crear la base de datos en caso de ser necesario
spring.jpa.hibernate.ddl-auto=update
spring.jpa.show-sql=true

package utn.tienda_libros;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class TiendaLibrosApplication{
	public static void main(String[] args){ SpringApplication.run(TiendaLibrosApplication.class, args);}
//Fin Ejercicio Anabel Alesci

//Ejercicio Matías Villa
#Conexión a MySQL

spring.datasource.url=jdbc:mysql://localhost:3306/tienda_libros_db?createDatabaseIfNotExist=true
spring.datasource.username=root
spring.datasource.password=admin
spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver

#Crear la base de datos en caso de ser necesario
spring.jpa.hibernate.ddl-auto=update
spring.jpa.show-sql=true

//Main
package utn.tienda_libros;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class TiendaLibrosApplication {

	public static void main(String[] args) {
		SpringApplication.run(TiendaLibrosApplication.class, args);
	}

}
//Fin Ejercicio Matías.

File: Node/Septiembre/Semana-18-al-22.js
// Ejercicio Jesús Mercado
// entorno.js

let nombre = process.env.NOMBRE || 'Sin nombre';
let web = process.env.MI_WEB || 'No tengo web';

console.log('Hola '+ nombre);
console.log('Mi web es: ' + web);

// Fin Ejercicio Jesús Mercado

//Ejercicio Gerardo Duckwitz
let nombre = process.env.NOMBRE || "Sin nombre";
let web = process.env.WEB || "No tengo web";
console.log("Hola" + saludo);
console.log("Mi web es: " + web);

//Fin ejercicio Gerardo Duckwitz

//Ejercicio de Yesica López

let nombre = process.env.NOMBRE || 'Sin nombre';
let web = process.env.MI_WEB || 'No tengo web';

console.log('Hola '+nombre);
console.log('Mi web es: '+web);

//Fin Ejercicio Yesica López
console.log('Mi web es: ' + web);


// Ejercicio Jose Remaggi
// entorno.js

let nombre = process.env.NOMBRE || 'Sin nombre';
let web = process.env.MI_WEB || 'No tengo web';

console.log('Hola '+ nombre);
console.log('Mi web es: ' + web);

// Fin Ejercicio Jose Remaggi

// Ejercicio Anabel Alesci
// entorno.js

let nombre = process.env.NOMBRE || 'Sin nombre';
let web = process.env.MI_WEB || 'No tengo web';

console.log('Hola '+ nombre);
console.log('Mi web es: ' + web);

// Fin Ejercicio Anabel Alesci

//Ejercicio Victoria zaccaro
let nombre = process.env.NOMBRE || 'Sin nombre';//si no existe la variable de entorno, se le asigna el valor 'Sin nombre'
let web = process.env.MI_WEB || 'No tengo web';//si no existe la variable de entorno, se le asigna el valor 'No tengo web'

console.log('Hola '+ nombre);//imprime en consola
console.log('Mi web es: ' + web);//imprime en consola
// Fin ejercicio victoria zaccaro

//Ejercicio Matías Villa

//Variable de entorno
//Ingresar informaciòn desde afuera hacia adentro
let nombre = process.env.NOMBRE || 'Sin nombre';
let web =   process.env.MI_WEB || 'No tengo web';
console.log('Hola ' + nombre);
console.log('Mi web es: '+web);


//en consola: $env:NOMBRE="CARLOS"; node entorno.js
//Se usa en mayusculas las variables de entorno porque así se hacia en linux
//Fin ejercicio Matías Villa. 
